users/models/profile.py

```diff
@@ -26,7 +26,7 @@ class Profile(models.Model):
 
     Django forms and serializer do not allow you to leave data empty.
     """
-    user = models.OneToOneField(User, on_delete=models.CASCADE,
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE,
```

Comment:
Generally speaking it would be better to use the [get_user_model](https://docs.djangoproject.com/en/2.1/topics/auth/customizing/#django.contrib.auth.get_user_model) helper here instead of direct import